net-libs/zeromq/zeromq-4.3.5-r2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="High-performance asynchronous messaging library"
+HOMEPAGE="https://zeromq.org/"
+SRC_URI="https://github.com/zeromq/libzmq/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0/5"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~arm64-macos ~x64-macos"
+IUSE="doc drafts intrinsics +libbsd +sodium static-libs test unwind"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	!elibc_Darwin? ( unwind? ( sys-libs/libunwind ) )
+	libbsd? ( dev-libs/libbsd:= )
+	sodium? ( dev-libs/libsodium:= )
+"
+DEPEND="
+	${RDEPEND}
+	!elibc_Darwin? ( sys-apps/util-linux )
+"
+BDEPEND="
+	virtual/pkgconfig
+	doc? (
+		app-text/asciidoc
+		app-text/xmlto
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.3.5-c99.patch
+)
+
+src_prepare() {
+	cmake_src_prepare
+
+	sed -i '/list(APPEND tests test_busy_poll)/d' CMakeLists.txt || die
```

Comment:
This doesn't seem to do anything,
https://github.com/zeromq/libzmq/blob/622fc6dde99ee172ebaa9c8628d85a7a1995a21d/CMakeLists.txt (no such line)
```
 * SED: the following did not cause any changes
 *     sed -i '/list(APPEND tests test_busy_poll)/d' CMakeLists.txt || die
 * no-op: -e /list(APPEND tests test_busy_poll)/d
```